hybrasyl/Login.cs

```diff
@@ -129,7 +130,7 @@ private void PacketHandler_0x03_Login(Client client, ClientPacket packet)
                     client.ConnectionId, name);
                 client.Redirect(redirect);
                 loginUser.Login.LastLogin = DateTime.Now;
-                loginUser.Login.LastLoginFrom = ((IPEndPoint) client.Socket.RemoteEndPoint).Address.ToString();
+                loginUser.Login.LastLoginFrom = ((IPEndPoint) client.Socket.RemoteEndPoint).Address.ToString();          

```

Comment:
Please do not add unnecessary whitespace.

Reply:
Same as below, not sure why this whitespace was added. Fixed.